app/models/sitemap.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+class Sitemap < ActiveRecord::Base
+  RECENT_SITEMAP_NAME = 'recent'
+  NEWS_SITEMAP_NAME = 'news'
+
+  class << self
+    def regenerate_sitemaps
+      names_used = [RECENT_SITEMAP_NAME, NEWS_SITEMAP_NAME]
+
+      names_used.each { |name| touch(name) }
+
+      count = indexable_topics.count
```

Comment:
I'm wondering if we able to rely on `Category#topic_count` here instead? That would be a more efficient query than trying to count every single topic.